// reactproject/js/scripts/index.js
export const tilesPacient = [
  {
    faIcon: 'fas fa-pencil-alt',
    description: 'Programează-te',
    backgroundImage: '../img/consultatie.png',
    link: '/#/formular-programare'
  },
  {
    faIcon: 'fas fa-calendar-alt',
    description: 'Programări efectuate',
    backgroundImage: '../img/appointment.png',
    link: '/#/lista-programari'
  },
  {
    faIcon: 'fas fa-cog',
    description: 'Setări',
    backgroundImage: '../img/setari.png',
    link: '/#/setari'
  },
  {
    faIcon: 'fas fa-sign-out-alt',
    description: 'Deconectare',
    backgroundImage: '../img/logout.png',
    link: '/#/login'
  }
];

export const tilesDoctor = [
  {
    faIcon: 'fas fa-pencil-alt',
    description: 'Programează-te',
    backgroundImage: '../img/consultatie.png',
    link: '/#/formular-programare'
  },
  {
    faIcon: 'fas fa-calendar-alt',
    description: 'Programări efectuate',
    backgroundImage: '../img/appointment.png',
    link: '/#/lista-programari'
  },
  {
    faIcon: 'fas fa-cog',
    description: 'Setări',
    backgroundImage: '../img/setari.png',
    link: '/#/setari'
  },
  {
    faIcon: 'fas fa-sign-out-alt',
    description: 'Deconectare',
    backgroundImage: '../img/logout.png',
    link: '/#/login'
  }
];
