// reactproject/js/component/login/index.js
import React, {Component} from 'react';
import Input from '../common/input/index';
import Panel from '../common/panel/index';
import Logo from '../common/logo/index';
import Button from '../common/button/index';
import './index.css'
import Redirect from "react-router-dom/es/Redirect";

class Login extends Component {

  constructor(props) {
    super(props);
    this.state = {
      register: false,
      recoverPass: false,
      logged: false
    }
  }

  setLogged = () => {
    this.setState({
      logged: true
    })
  };

  setRedirectRegister = () => {
    this.setState({
      register: true
    })
  };

  setRedirectRecoverPass = () => {
    this.setState({
      recoverPass: true
    })
  };

  render() {
    if (this.state.logged) {
      return <Redirect to='main'/>
    }
    else if (this.state.register) {
      return <Redirect to='/register'/>
    }
    else if(this.state.recoverPass) {
      return <Redirect to='/recover'/>
    }
    else {
      return (
          <div className="project-login">
            <Panel>
              <Logo/>
              <div className='project-input-combo'>
                <Input
                    label="Username"
                    type="text"
                />
                <Input
                    label="Password"
                    type="password"
                />
              </div>
              <div className='project-button-combo'>
                <Button label='Login' onClick={this.setLogged}/>
                <Button label='Register' onClick={this.setRedirectRegister}/>
                <Button label='Recover Password' onClick={this.setRedirectRecoverPass}/>
              </div>
            </Panel>
          </div>
      )
    }
  }
}

export default Login
